setup.py

```diff
@@ -6,7 +6,9 @@
     from pypandoc import convert
     read_md = lambda f: convert(f, 'rst')
 except ImportError:
-    print("warning: pypandoc module not found, could not convert Markdown to RST")
+    print(
+        "warning: pypandoc module not found, could not convert Markdown to RST"
+    )
```

Comment:
Same as above. I only made this change to satisfy my own execution of `flake8` but in retrospect Travis CI never complained. I'll back this out.